src/main/resources/db/migration/claimstore/V2019_04_12_1414__schema_change_for_claim_submission_operation_indicator.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE claim
+  ADD COLUMN submission_operation_ind JSONB NULL;
```

Comment:
How about `submission_operation_indicators`?

Reply:
@sabahirfan sure

Reply:
Checked. The latest Postgress allows 63 characters. So will change it